nextcloudappstore/core/static/assets/js/ui/app-detail.js

```diff
@@ -27,8 +27,16 @@
                         ratingTarget.appendChild(template);
                     });
                 } else {
-                    let templateNoComments = document.importNode(ratingTemplateNoComments.content, true);
-                    ratingTarget.appendChild(templateNoComments);
+                    let langCode = global.id('comment_display_language_code');
+                    let options = Array.from(langCode.options)
+                                    .filter( (o) => { return o.value === fallbackLanguageCode; } );
```

Comment:
you can completely drop the block here and replace it with an expression, the following are equivalent:

```js
(o) => { return o.value === fallbackLanguageCode; }
(o) => o.value === fallbackLanguageCode
```